platforms/documentation/docs/src/docs/userguide/dep-man/declaring-repositories/supported_metadata_formats.adoc

```diff
@@ -0,0 +1,162 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[declaring-repositories]]
+= Metadata Formats
+
+Metadata refers to the information associated with a module or dependency that describes its characteristics, relationships, and requirements.
+
+This metadata includes details such as:
+
+1. **Module Identity**: The group, name, and version (GAV) coordinates that uniquely identify the module.
+2. **Dependencies**: A list of other modules or libraries that this module depends on, including their versions and configurations.
+3. **Variants**: Different forms of the module (e.g., API, implementation, runtime) that can be consumed or resolved based on the context.
+4. **Artifacts**: The actual files (like JARs, ZIPs, etc.) produced by the module, which may include compiled code, resources, or documentation.
+5. **Capabilities**: Describes the functionality or features that a module provides or consumes, helping to avoid conflicts when different modules provide the same capability.
```

Comment:
I think we should either include attributes here too, or honestly just gloss over capabilities and not include this bullet point